lib/views/screens/loginPage.dart

```diff
@@ -0,0 +1,42 @@
+import 'package:flutter/material.dart';
+
+class loginPage extends StatelessWidget {
```

Comment:
Convert to Stateful widget, we will most probably need Stateful widget to manage Login State